cico_functions.sh

```diff
@@ -115,10 +115,17 @@ function build_and_push() {
 # arbitrary user patch
 function build_and_push_release() {
   echo "CICO: building release '${TAG}' version of devfile registry"
-  docker build -t ${IMAGE} -f ${DOCKERFILE_PATH} . \
-    --build-arg PATCHED_IMAGES_REG=${REGISTRY} \
-    --build-arg PATCHED_IMAGES_ORG=${ORGANIZATION} \
-    --build-arg PATCHED_IMAGES_TAG=${TAG}
+  if [ "$TARGET" == "rhel" ]; then
+    # Do not update registry and tags on RHEL releases, since we don't build
+    # patched images there.
+    docker build -t ${IMAGE} -f ${DOCKERFILE_PATH} . \
+      --build-arg PATCHED_IMAGES_TAG=${TAG}
```

Comment:
@amisevsk just to confirm, for rhel build the following argument will be used by default?

> ARG PATCHED_IMAGES_REG="quay.io"
> ARG PATCHED_IMAGES_ORG="eclipse"

Maybe we can simply remove ` if [ "$TARGET" == "rhel" ]` and override only tag in general ?

Reply:
That would be another option; the reason it's currently done the way it is is to support the case of building a registry top to bottom for your own org. I guess that's less useful in our specific CI though, since really only we would use it.